Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -1337,6 +1337,59 @@ public unsafe void Actions_CanTriggerBindingResolutionOnAction_FromCallback(bool
         Assert.That(action.controls, Is.EquivalentTo(new InputControl[] {gamepad.leftStick, mouse.delta}));
     }
 
+    // ReSharper disable once ClassNeverInstantiated.Local
+    private class ConstantFloatTestProcessor : InputProcessor<float>
+    {
+        public const float EXPECTED_VALUE = 0.5f;
+        public override float Process(float value, InputControl control)
+        {
+            return EXPECTED_VALUE;
+        }
+    }
+
+    [Test]
+    [Category("Actions")]
+    public void Actions_ActiveBindingsHaveCorrectBindingIndicesAfterBindingResolution()
```

Comment:
Nice to see a new test but was a little surprised as it doesn't seem related to re-enabling tests.

Reply:
Ah I see it in the description - thanks